cmd/app.go

```diff
@@ -81,6 +82,16 @@ func (sub *substation) SetConfig(r io.Reader) error {
 	return nil
 }
 
+// GetConfig retreives the configuration of the app.
+func (sub *substation) GetConfig() (io.Reader, error) {
```

Comment:
style: for getters, we don't prefix with Get, so i think this should just be `Config`